src/Fable.Transforms/Rust/Fable2Rust.fs

```diff
@@ -449,7 +453,14 @@ module TypeInfo =
 
     let rec isCloneableExpr (com: IRustCompiler) typ e =
         match e with
-        | Fable.Get _
+        | Fable.Get(_, kind, _, _) ->
+            match kind with
+            | Fable.OptionValue _
+            | Fable.UnionField _
+            | Fable.FieldGet _
+            | Fable.ListHead _
+            | Fable.ListTail _ -> true
+            | _ -> false
```

Comment:
Can you list the other kinds in the `false` category instead of a `_`, just so it's a complete pattern match?

Reply:
Of course. All done.